lib/Models/WebProcessingServiceCatalogFunction.js

```diff
@@ -53,6 +56,13 @@ function WebProcessingServiceCatalogFunction(terria) {
      */
     this.identifier = undefined;
 
+    /**
+     * Gets or sets whether to use key value pairs (KVP) embedded in Execute URL, or whether to make a POST request
+     * with XML data.
+     * @type {Boolean}
+     */
+    this.useKvp = undefined;
```

Comment:
I noticed this is `knockout.track`'d in the group, but not in the function here.  It should be here too I think.